media/inf/nettcpip.inf

```diff
@@ -254,7 +254,8 @@ HKR,"Parameters\PersistentRoutes",,0x00000010
 [tcpip_AddReg]
 HKR,"Parameters","DataBasePath",0x00020000,"%SystemRoot%\System32\drivers\etc"
 HKR,"Parameters","Domain",0x00000000,""
-HKR,"Parameters","Hostname",0x00000000,"ROSHost"
+HKR,"Parameters","Hostname",0x00000000,"ROSHOST"
+HKR,"Parameters","NV Hostname",0x00000000,"ROSHOST"
```

Comment:
original patch changed this to "COMPUTERNAME", should be better.

Reply:
How `COMPUTERNAME` is better? It would then match the name of env variable, which is not very well tbh.

Reply:
out of curiosity, I wonder what the default computer name is when running Windows PE.

Reply:
`MINWIN`, iirc

Reply:
even the Windows 2003 PE?

Reply:
OK I was able to answer my own question:
![image](https://github.com/reactos/reactos/assets/1969829/3eadb1bc-cf25-4649-b4f9-695c5a4aee05)

1. WinPE starts and uses the hardcoded computername `MININT-JVC` (that can be seen from the `COMPUTERNAME` environment variable in cmd.exe)
2. Then it starts the `factory -winpe` program, which dynamically changes the computer name, as can be seen in the registry (the part after `MININT-` is randomly-generated, like what we do during our ReactOS install). When one re-runs manually the `factory -winpe` command in the opened cmd, this reconfigures the environment and changes again the value stored in the registry.
3. Both `ActiveComputerName` and `ComputerName` keys/values in the registry contain the same string.